jobscheduler/scheduler-tool/src/java/org/sakaiproject/tool/app/scheduler/SchedulerTool.java

```diff
@@ -788,30 +788,28 @@ public String processCreateTrigger()
       LOG.error("Scheduler is down!");
       return "error";
     }
-    try
-    {
-    	JobDetail
-    		jd = selectedJobDetailWrapper.getJobDetail();
-
-    	Trigger trigger = TriggerBuilder.newTrigger()
-    	        .withIdentity(jd.getKey().getName(), Scheduler.DEFAULT_GROUP)
-    	        .withSchedule(CronScheduleBuilder.cronSchedule(triggerExpression))
-    	        .build();
-    	        
+    try {
+        JobDetail jobDetail = selectedJobDetailWrapper.getJobDetail();
+
+        Trigger trigger = TriggerBuilder.newTrigger()
+                .withIdentity(triggerName, Scheduler.DEFAULT_GROUP)
+                .withSchedule(CronScheduleBuilder.cronSchedule(triggerExpression))
+                .forJob(jobDetail.getKey())
```

Comment:
Is this the only addition here? It looks like the rest of this is just variable name changing.